### DISCUSSION TITLE
Category editing screen is empty

### DISCUSSION THREAD

**Discussion by xaver-ius** [craigk5n/webcalendar#449] [Q&A] [answered]
When I click -> Settings -> Categories I get an empty screen. I can go back to the calendar view.
I'm using WebCalendar V1.9.10 on my homepage.strato.de with Firefox 120.0.1 on Windows 10, 64Bit.
I have activated the categories in the settings, upload icons activation doesn't matter.
Is there any idea what I'm doing wrong?

**Comment by xaver-ius** [comment, marked as answer]
I added a directory /wc-icons and now I can edit categories :-)

**Reply by craigk5n** [reply]
Note that in v1.9.12 we get rid of the wc-icons directory and move the icon images into the database.  So, this should not be a problem with v1.9.12 and later (once it is released).
